src/components/pages/Mediation/MediationContainer.js

```diff
@@ -31,6 +31,7 @@ export const mapStateToProps = (state, ownProps) => {
 
 export const mapDispatchToProps = dispatch => {
   return {
+    dispatch,
```

Comment:
```suggestion
    resetForm: () => resetForm(),
```
Avec l'import qui va bien.
Ce qui fait que tu ne crées pas de coupable entre le store et le composant graphique.